src/index.html

```diff
@@ -245,13 +245,10 @@ <h2>最新バージョン</h2>
 						</div>
 					</div>
 
-					<div class="row flex-grow-1 plugins-table">
+					<div class="row flex-grow-1 plugins-table my-2">
 						<div class="col">
```

Comment:
I think the wider table is the better.

So, it can be:
```suggestion
					<div class="row flex-grow-1 plugins-table">
						<div class="col g-0">
```